src/book.js

```diff
@@ -563,122 +565,95 @@ export class Book {
    */
   draw_spine_marks(curPage, sigDetails, position) {
     const w = 5;
+    let startX, startY, endX, endY;
+
+    if (sigDetails.isSigStart) {
+      startX = position.spineMarkTop[0];
+      startY = position.spineMarkTop[1];
+      endX = position.spineMarkTop[0];
+      endY = position.spineMarkTop[1];
+    } else {
+      startX = position.spineMarkBottom[0];
+      startY = position.spineMarkBottom[1];
+      endX = position.spineMarkBottom[0];
+      endY = position.spineMarkBottom[1];
+    }
+
```

Comment:
Would love to see these as array destructuring (ie, `[startX, startY] = position.spineMarkTop`

Reply:
changed

Reply:
This isn't going to work as written, because the array items aren't arrays themselves, so you can't destructure them. I'm also not 100% sure that this matches the original behavior, though this is more because the original setup is kind of confusing to parse through - I will give it another pass and let you know though.

Reply:
Okay, so the way this should go is
````
draw_spine_marks(curPage, sigDetails, position) {
  const w = 5;
  let startX, startY, endX, endY;
    if (sigDetails.isSigStart) {
      [startX, startY] = position.spineMarkTop;
      [endX, endY] = position.spineMarkTop;
    } else {
      [startX, startY] = position.spineMarkBottom;
      [endX, endY] = position.spineMarkBottom;
    }

    if (position.rotation == 0) {
      startX -= w/2;
      endX += w/2 
    } else if (sigDetails.isSigStart) {
      startY -= w;
      endY += w;
    } else {
      startY -= w/2;
      endY += w/2;
    }
}
````
idk why one mark is longer than the other, but that's what's in the current code.